jsonrpcbase/main.py

```diff
@@ -53,11 +55,9 @@ def send_message(msg):
 """
 import json
 import logging
-import types
-
-import six
+import jsonschema
 
-DEFAULT_JSONRPC = '2.0'
+DEFAULT_JSONRPC = (2, 0)
```

Comment:
Before they were using an integer of the version like `20`, but I think a tuple makes more sense (eg `1.11` and `11.1` would be ambiguous as an integer). Plus, tuples are comparable in python, so you can do `(1,1) < (2,0)`